azure-pipelines.yml

```diff
@@ -230,7 +230,7 @@ stages:
             CoverageArtifactName: 'DotNetCoverageNet7'
             TestResultsArtifactName: 'DotNetTestResultsNet7'
             ProjectFilePath: 'tests\SonarAnalyzer.Test\SonarAnalyzer.Test.csproj'
-            FrameworkMoniker: 'net7.0'
```

Comment:
I will update the build configuration in a separate PR (when I move the tests from the dedicated project to net 8 to the general one).